yang/ietf-ntw-energy-saving.yang

```diff
@@ -52,7 +59,103 @@ module ietf-ntw-energy-saving {
     reference
       "RFC XXXX: YANG Data Models for Energy Saving Management";
   }
+  /*
+  Identity
+  */
+  identity energy-saving-power-state {
+    description
+      "Base identity for power state.";
+    reference
+      "RFC 7326: Energy Management Framework";
+  }
+
+  identity off-state {
+    base energy-saving-power-state;
+    description
+      "Indicates that the component typically
+       requires a complete boot when awakened.";
+    reference
+      "RFC 7326: Energy Management Framework";
+  }
 
+  identity sleep-state {
+    base energy-saving-power-state;
+    description
+      "Indicates that a component with energy management support
+       is not functional but immediately available such as
+       wake up mechanism.";
+    reference
+      "RFC 7326: Energy Management Framework";
+  }
+
+  identity low-power-state {
+    base energy-saving-power-state;
+    description
+      "Indicates that some components with energy management
+       support are available and can take measures to use less
+       energy.";
+    reference
+      "RFC 7326: Energy Management Framework";
+  }
+
+  identity full-power-state {
+       base energy-saving-power-state;
+       description
+         "Indicates that all components with energy management
+          support are available and may use maximum power.";
+       reference
+         "RFC 7326: Energy Management Framework";
+  }
+  /*
+   Grouping
+  */   
+  grouping energy-consumption-data {
```

Comment:
We don't need to define this one as it is part of the ietf-energy-saving-common

Reply:
OK